src/experimental/zookeeper/zookeeper/templates/svc.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "zookeeper.fullname" . }}-svc
+  labels:
+    app.kubernetes.io/name: {{ include "zookeeper.name" . }}
+    helm.sh/chart: {{ include "zookeeper.chart" . }}
+    # app.kubernetes.io/instance: {{ .Release.Name }}
+    # app.kubernetes.io/managed-by: {{ .Release.Service }}
+spec:
+  type: {{ .Values.service.type }}
+  ports:
+    - port: {{ .Values.service.clientPort }}
+      targetPort: zk-c-tport
+      protocol: TCP
+      name: zk-c-port
+    - port: {{ .Values.service.followPort }}
```

Comment:
Client ports should be enough, and follower/election ports should not be exposed. Nodes can communicate with each other by stateful endpoint.